pkgs/top-level/all-packages.nix

```diff
@@ -31755,6156 +31757,4 @@ with pkgs;
 
   sonic-pi = libsForQt5.callPackage ../applications/audio/sonic-pi { };
 
-  stag = callPackage ../applications/misc/stag {
```

Comment:
Not sure what happened but this change shouldn't be here.

Reply:
Yeah I have no idea what I did in that commit, just fixed it all haha